### PR TITLE
Test of DemoProject in WdIO

### PR DESCRIPTION
**Pull request by oscargarcia23**
PR of the tests of Demo Project in WDIO for Jope to review

These tests are:
- Display Inicio
- Display Servicios (all of them)
- Display Portaflio
- Display Conocenos
- Display Empleos
- Display Blog

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,8171 @@
+{
+  "name": "phase3demoproject",
```
Comment (joseperez2596) [inline package-lock.json]: This was my fault, but this file should not be included.

File: pagesGBH/GeneralActions.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  get ConfigRespMenu () { return $('#menu-item-360') },
+  get InstalacionCableadoMenu () { return $('#menu-item-357') },
+  get ConfigWifiMenu () { return $('#menu-item-358') },
+  get CertificCableMenu () { return $('#menu-item-356') },
+  get PortafolioMenu () { return $('#menu-item-62') },
+  get ConocenosMenu () { return $('#menu-item-65') },
+  get EmpleosMenu () { return $('#menu-item-126') },
+  get BlogMenu () { return $('#menu-item-20') }
+}
+export function openservicios () {
```
Comment (joseperez2596) [inline pagesGBH/GeneralActions.js]: Please define a capitalization standard for your project and adjust it to it.

File: tests/specs/AsesoriaTest.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { openGBHBrowser } from '../../pagesGBH/Pagelink'
+import { openservicios, ClickAsesoriaUI } from '../../pagesGBH/GeneralActions'
+var expect = require('expect.js')
+
+describe('Check Inicio page is correct', () => {
+  it('should tell if the page is Home of GBH webplace', () => {
+    openGBHBrowser()
+    var browserTitle = browser.getTitle().toString()
+    browser.pause(3000)
+    expect(browserTitle).to.contain('Servicios informaticos')
```
Comment (joseperez2596) [inline tests/specs/AsesoriaTest.js]: This check should be on a check action on that page's action files.

File: tests/specs/AsesoriaTest.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('Click the Asesoria tab and check if the title is correct', () => {
+  it('should Hover to the Servicios tab and click Asesoria tab ', () => {
+    openservicios()
+    ClickAsesoriaUI()
+  })
+
+  it('and check if the title is correct', () => {
+    var browserTitle = browser.getTitle().toString()
+    expect(browserTitle).to.contain('Asesoría de interfaz')
```
Comment (joseperez2596) [inline tests/specs/AsesoriaTest.js]: All checks should be on the page's actions file.

File: pagesGBH/Pagelink.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export function openGBHBrowser () {
```
Comment (joseperez2596) [inline pagesGBH/Pagelink.js]: This function starts with lower case.

Comment (oscargarcia23) [reply pagesGBH/Pagelink.js]: I missed that one

**Pull request merged**